composer.json

```diff
@@ -45,21 +44,29 @@
         "post-install-cmd": "./vendor/bin/run drupal:site-setup",
         "post-update-cmd": "./vendor/bin/run drupal:site-setup"
     },
-    "repositories": [
-        {
+    "repositories": {
+        "drupal": {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
+        "ec-europa/oe-poetry-client": {
+            "type": "git",
+            "url": "https://github.com/GilNovacomm/oe-poetry-client.git"
+        },
+        "openeuropa/oe_multilingual": {
+            "type": "git",
+            "url": "https://github.com/GilNovacomm/oe_multilingual.git"
```

Comment:
DO NOT Merge. These repositories were for testing.

Reply:
Rather get write permissions and create a branch within the repo instead